.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: false # enable container-based infrastructure
+
+addons:
+  apt:
+    sources:
+      - deadsnakes # source required so it finds the package definition below
+    packages:
+      # packahe list:
+      # https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+      - python3.5 python3.5-dev python3.5-venv check libglew-dev
```

Comment:
If we're not going to invoke tests on dependencies (that should be done by their own CI), `check` is unneded.